qui/clipboard.py

```diff
@@ -79,10 +85,10 @@ def _copy(self, vmname: str = None):
 
         size = clipboard_formatted_size()
 
-        body = _("Clipboard contents fetched from qube: <b>'{0}'</b>\n"
-                 "Copied <b>{1}</b> to the global clipboard.\n"
-                 "<small>Press Ctrl+Shift+V in qube to paste to local"
-                 "clipboard.</small>").format(vmname, size)
+        body = _(f"Clipboard contents fetched from qube: <b>'{vmname}'</b>\n"
+                 f"Copied <b>{size}</b> to the global clipboard.\n"
+                 f"<small>Press {self.gtk_app.paste_shortcut} in qube "
+                 "to paste to local clipboard.</small>")
```

Comment:
`f""`applied this way isn't compatible with gettext. Please leave it as a `format()` call for now (or similar alternative).

Reply:
fixed